### DISCUSSION TITLE
How to specifiy StimulusEpoch.stimulus_start_time

### DISCUSSION THREAD

**Discussion by bruno-f-cruz** [AllenNeuralDynamics/aind-data-schema#579] [Q&A]
StimulusEpoch class https://github.com/AllenNeuralDynamics/aind-data-schema/blob/9967a02777713a02cb07f826d7a6539e9fcf57ef/src/aind_data_schema/stimulus.py#L163

states that `stimulus_start_time`  can be the same as the session start time. However, the types are different and they don't serialize from the same data types. For instance, while both `Stream` and `Session` serialize from `DateTime`, `StimulusEpoch` uses `Time` as its expected type. Is there any reason for these to not be the same? 

https://github.com/AllenNeuralDynamics/aind-data-schema/blob/9967a02777713a02cb07f826d7a6539e9fcf57ef/src/aind_data_schema/session.py#L260

https://github.com/AllenNeuralDynamics/aind-data-schema/blob/9967a02777713a02cb07f826d7a6539e9fcf57ef/src/aind_data_schema/session.py#L225

What is the correct way to use this field? thanks!

**Comment by saskiad** [comment]
There's an open issue on this - they should both be datetime. We'll fix it asap.

**Comment by bruno-f-cruz** [comment]
Duplicate of  #557
